cli/config_file.go

```diff
@@ -156,6 +159,9 @@ func SettingsFileReadWrite(state *Settings) *Settings {
 	// Settings Validation steps
 
 	// TODO(c3ppc3pp): Check for available ports
```

Comment:
Please remove the TODO if you are ready with the feature.

Reply:
done